src/pages/NotFound.tsx

```diff
@@ -1,30 +1,76 @@
-import { AccentText, CardParagraph, HeaderText } from 'components/Text'
-import { space } from 'classnames/tailwind'
-import { useNavigate } from 'react-router-dom'
+import { useEffect } from 'preact/hooks'
 import Button from 'components/Button'
-import Card from 'components/Card'
-import CardSection from 'components/CardSection'
+import classNamesToString from 'helpers/classNamesToString'
+import classnames, {
+  alignItems,
+  display,
+  dropShadow,
+  flexDirection,
+  fontFamily,
+  fontSize,
+  fontWeight,
+  gap,
+  justifyContent,
+  margin,
+  textAlign,
+  textColor,
+  width,
+} from 'classnames/tailwind'
+
+const container = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  alignItems('items-center'),
+  textAlign('text-center'),
+  gap('gap-y-4')
+)
+const octoBlock = classnames(
+  display('flex'),
+  flexDirection('flex-col', 'xl:flex-row'),
+  alignItems('items-center'),
+  justifyContent('justify-center'),
+  gap('gap-x-40', 'gap-y-14')
+)
+const textStyles = classnames(width('w-fit', 'xl:w-max'), margin('mt-7'))
+const strokeText = classNamesToString(
+  classnames(
+    textColor('text-transparent'),
+    dropShadow('drop-shadow-secondary'),
+    fontSize('text-8xl'),
+    fontFamily('font-primary'),
+    fontWeight('font-bold')
+  ),
+  'stroke-text-secondary'
+)
+const imageStyles = width('w-80', 'xl:w-full')
+const displayFromMd = display('hidden', 'xl:block')
 
-const container = space('space-y-4')
 export default function () {
-  const navigate = useNavigate()
+  document.title = '🐙 OCTOCORP'
+
+  useEffect(() => {
+    document.getElementsByTagName('html')[0].setAttribute('bg-theme', '404')
+
+    return () => {
+      document.getElementsByTagName('html')[0].setAttribute('bg-theme', '')
+    }
+  })
```

Comment:
Maybe it's better to move to the hook? It might be useful in the long term

Reply:
Let's do this